src/igvfd/schemas/biomarker.json

```diff
@@ -0,0 +1,75 @@
+{
+    "title": "Biomarker",
+    "description": "Schema for submitting a biomarker.",
+    "$id": "/profiles/biomarker.json",
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "type": "object",
+    "required": [
+        "name",
+        "quantification"
+    ],
+    "identifyingProperties": [
+        "uuid",
+        "name_quantification"
+    ],
+    "additionalProperties": false,
+    "mixinProperties": [
+        {
+            "$ref": "mixins.json#/basic_item"
+        }
+    ],
+    "dependentSchemas": {},
+    "properties": {
+        "schema_version": {
+            "default": "1"
+        },
+        "name": {
+            "title": "Name",
+            "description": "The biomarker name.",
+            "comment": "Only admins are allowed to set this value.",
+            "type": "string",
+            "permission": "import_items"
+        },
+        "classification": {
+            "title": "Biomarker Classification",
+            "description": "Sample specific biomarker.",
+            "type": "string",
+            "enum": [
+                "cell surface protein",
+                "marker gene"
+            ]
+        },
+        "quantification": {
+            "title": "Quantification",
+            "description": "The biomarker association to the biosample, disease or other condition.  This can be the absence of the biomarker or the presence of the biomarker in some low, intermediate or high quantity.",
+            "comment": "Only admins are allowed to set this value.",
+            "type": "string",
+            "enum": [
+                "negative",
+                "positive",
+                "low",
+                "intermediate",
+                "high"
+            ],
+            "permission": "import_items"
+        },
+        "synonyms": {
+            "title": "Synonyms",
+            "description": "Alternate names for this biomarker.",
+            "comment": "These can vary depending on the type of biomarker.",
```

Comment:
I think this comment could be removed - not sure it provides any additional information

Reply:
done